internal/backend/oauth/oauth.go

```diff
@@ -59,6 +61,7 @@ func New(l *zap.Logger, vars sdkservices.Vars) sdkservices.OAuth {
 	l.Debug("Atlassian base URL for OAuth", zap.String("url", atlassianBaseURL))
 
 	// Determine GitHub base URL (to support GitHub Enterprise Server, i.e. on-prem).
+	// TODO: Add support for custom OAuth.
```

Comment:
Create a ticket, and specify the ID here. Mention this PR in that ticket as a context link.